ci/base-tests.sh

```diff
@@ -1,16 +1,18 @@
+#!/usr/bin/env bash
 set -ex
 
 echo "Running clippy base tests"
 
 PATH=$PATH:./node_modules/.bin
-if [ "$TRAVIS_OS_NAME" == "linux" ]; then
-  remark -f *.md -f doc/*.md > /dev/null
+if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then
+  remark -f ./*.md -f doc/*.md > /dev/null
 fi
 # build clippy in debug mode and run tests
 cargo build --features "debugging deny-warnings"
 cargo test --features "debugging deny-warnings"
 # for faster build, share target dir between subcrates
-export CARGO_TARGET_DIR=`pwd`/target/
+CARGO_TARGET_DIR=$(pwd)/target/
+export CARGO_TARGET_DIR
```

Comment:
Why?

Reply:
It is to avoid masking return values of pwd.